Bigtable/src/DataClient.php

```diff
@@ -118,16 +118,17 @@ public function __construct($instanceId, $tableId, array $config = [])
      * $dataClient->mutateRows([$rowMutation]);
      * ```
      * @param array $rowMutations array of RowMutation object.
+     * @param array $options [optional] Configuration options.
      * @return void
      * @throws ApiException|BigtableDataOperationException if the remote call fails or operation fails
      */
-    public function mutateRows(array $rowMutations)
+    public function mutateRows(array $rowMutations, array $options = [])
     {
         $entries = [];
         foreach ($rowMutations as $rowMutation) {
             $entries[] = $rowMutation->getEntry();
         }
-        $responseStream = $this->bigtableClient->mutateRows($this->tableName, $entries, $this->options);
+        $responseStream = $this->bigtableClient->mutateRows($this->tableName, $entries, $this->options + $options);
```

Comment:
The local value of `$options` should, I think, take precedence over `$this->options`, right? In other words, if `$options` and `$this->options` contain the same key, the value from `$options` should take precedence.

If that is correct, the order of all these append operations should be reversed to `$options + $this->options`.

Reply:
The general idea here being what we are accepting options to be proxied through to the GAPIC is good enough for me right now. We can touch on these minor details as we iterate over this once we have our gameplan for removing the connection layer in place.